src/core/interval.test.js

```diff
@@ -75,14 +75,6 @@ describe("src/core/interval", () => {
       const thunk = () => intervalSequence([i0, i1]);
       expect(thunk).toThrowError("NaN");
     });
-    it("resists mutation of the intervals", () => {
```

Comment:
Why was this deleted?

Reply:
It's checked statically now by flow

Reply:
Update: I added this test back in with an error suppression, since we want to preserve the functionality.